### PR TITLE
Add experimental admonition

### PR DESCRIPTION
**Pull request by MartinMinkov**
Preview:

image

### CODE REVIEW

File: src/scss/utilities/_variables.scss [hunk 1]
```diff
@@ -13,3 +13,4 @@ $mina-black: #2d2d2d;
 $mina-white: #ffffff;
 $mina-error: #e93939;
 $mina-purple: #5362c8;
+$mina-lavender: #cab2ff;
```
Comment (ymekuria) [inline src/scss/utilities/_variables.scss:16]: I haven't seen this color yet. I like it!

File: package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
     "start": "docusaurus start",
     "dev": "ALGOLIA_APP_ID='.' ALGOLIA_SEARCH_API_KEY='.' docusaurus start",
     "build": "docusaurus build",
-    "swizzle": "docusaurus swizzle",
+    "swizzle": "docusaurus swizzle --typescript",
+    "swizzle-list": "docusaurus swizzle --list",
```
Comment (ymekuria) [inline package.json:11]: This is a nice shortcut. Thanks for adding this.

**Review by ymekuria — APPROVED** [review]
Looks great!

**Comment by barriebyron** [comment]
great, @MartinMinkov  let's do it!

**Comment by barriebyron** [comment]
docs are updated and look great https://github.com/o1-labs/docs2/pull/587 

check it out https://docs.minaprotocol.com/zkapps/o1js/recursion

**Pull request merged**